media-libs/libffado/libffado-2.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+[[ "${PV}" = "9999" ]] && inherit subversion
+inherit eutils python-single-r1 scons-utils toolchain-funcs udev multilib-minimal
+
+DESCRIPTION="Driver for IEEE1394 (Firewire) audio interfaces"
+HOMEPAGE="http://www.ffado.org"
+
+RESTRICT="mirror"
+if [ "${PV}" = "9999" ]; then
+	ESVN_REPO_URI="http://subversion.ffado.org/ffado/trunk/${PN}"
+	SRC_URI=""
+	KEYWORDS=""
+else
+	SRC_URI="http://www.ffado.org/files/${P}.tgz"
+	KEYWORDS="~amd64 ~ppc ~x86"
+fi
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+IUSE="debug qt4 +test-programs"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="dev-cpp/libxmlpp:2.6[${MULTILIB_USEDEP}]
+	dev-libs/dbus-c++
+	dev-libs/libconfig[cxx,${MULTILIB_USEDEP}]
+	media-libs/alsa-lib
+	media-libs/libiec61883[${MULTILIB_USEDEP}]
+	!<media-sound/jack-audio-connection-kit-0.122.0:0
+	!<media-sound/jack-audio-connection-kit-1.9.9:2
```

Comment:
@marcan are these jack versions really incompatible? Or is it just something you didn't validate?
Asking because in the gentoo tree `jack-audio-connection-kit-0.121.3-r1` is the only one marked as stable.